html/browsers/the-window-object/window-indexed-properties.html

```diff
@@ -68,4 +68,14 @@
     assert_false(idx in proto, idx + " in proto");
   });
 });
+test(t => {
+  Object.defineProperty(Window.prototype, 2, { set: t.unreached_func("Window.prototype[2] setter should be unreachable!") });
+  window[2] = 2;
+  assert_false(window.hasOwnProperty(2));
+  Object.defineProperty(EventTarget.prototype, 42, { set: t.unreached_func("EventTarget.prototype[42] setter should be unreachable!") });
+  assert_false(Reflect.set(window, 42, "foo"));
```

Comment:
Is there a particular reason to use `Reflect.set` here?

Reply:
Can you use a normal set here, and put the `Reflect` check in a separate `test()` below?